app/controllers/api/v2/override_values_controller.rb

```diff
@@ -88,6 +88,7 @@ def find_override_values
 
       def find_override_value
         @override_value = LookupValue.find_by_id(params[:id])
+        @override_value ||= @override_values.friendly.find(params[:id])
```

Comment:
Sorry for the delay, shouldn't `@override_values` be changed here to `LookupValue` again? `@override_values` is set to `@smart.lookup_values.paginate(paginate_options)` - which means that we're not really searching all LookupValues.

Reply:
Thanks for the feedback, very true about pagination I had overlooked that, some extra logic is still needed vs what you sugguest however as need to scope the lookupvalue by Smart variable/class param otherwise it's not unique. Will submit a proposal. 

In terms of tests, the override test at the moment use to_param i.e. https://github.com/theforeman/foreman/blob/develop/test/controllers/api/v2/override_values_controller_test.rb#L45 which defaults to then using the match string anyway due to friendly, I can split this out to explicitly use id & match based calls though?